saleor/discount/migrations/0065_vouchercode_created_at_check.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.23 on 2023-11-17 08:08
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("discount", "0064_alter_promotionrule_catalogue_predicate"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql=(
```

Comment:
I would suggest to simplify that query to just:
`ALTER TABLE discount_vouchercode ADD COLUMN IF NOT EXISTS created_at TIMESTAMP;`

I now see one problem here, I'm not convinced if we should set default on the database, it's a function so the migration will be costly here. Do we know the impact of that table? (cause the data migration with batches may be needed to propagate the default value)